vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -1280,13 +1280,24 @@ public Component getListCellRendererComponent(
       // DO NOT pass value to super.getListCellRendererComponent()
       // It is incredibly inefficient for GamePieces and is not needed
       // since we overwrite the label text anyway.
+      String lineNumber;
+      if (PieceDefiner.prototypeName.isEmpty()) {
+        final int index1 = index + 1;
+        lineNumber = " ".repeat(index1 < 10 ? 2 : index1 < 100 ? 1 : 0) + index1 + ". ";
+      }
+      else {
+        lineNumber = (index == 0 ? "" : " ".repeat(index < 10 ? 2 : index < 100 ? 1 : 0) + index + ". ");
+      }
```

Comment:
Given that the font used in the PieceDefiner is variable width, this code doesn't really have any effect, it does not cause the single digit and two digit numbers to line up. I would just removed all the leading spaces and just go with left justified numbers.